app/adapters/whats_app_adapter/three_sixty_dialog_inbound.rb

```diff
@@ -19,20 +19,18 @@ def consume(organization, whats_app_payload)
       @quote_reply_message_id = message_payload.dig(:context, :id)
       @text = initialize_text
 
-      if ephemeral_data?
-        handle_ephemeral_data
-        return
-      end
+      return if ephemeral_data_handled?
```

Comment:
That's better than duplicating the long `if-else` statement, I'd say. :shrug:

Reply:
For sure.